### PR TITLE
UI scaffolding initial version as starter.

### PR DESCRIPTION
**Pull request by HedayatAbedijoo**

### CODE REVIEW

File: ui-hedayat-suggetion/frontend/src/index copy.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import "./view/header/page-header";
```
Comment (harlantwood) [inline ui-hedayat-suggetion/frontend/src/index copy.ts:1]: Do you want to include this “index copy” file?

Comment (harlantwood) [inline ui-hedayat-suggetion/frontend/src/index copy.ts:1]: Do you want to include this “index copy” file?

Comment (HedayatAbedijoo) [reply ui-hedayat-suggetion/frontend/src/index copy.ts:1]: no no , we need to delete this,
thanks for review.

**Comment by nphias** [comment]
I am not not seeing the UI getting a 404.. (with back end running)

**Comment by HedayatAbedijoo** [comment]
the issue is fixed. try it again.

**Pull request merged**